src/components/Events.jsx

```diff
@@ -30,7 +30,9 @@ const Events = () => {
           />
         ))}
       </div>
-      <Button text={"See Calendar"} />
+      <div className="hover:invert">
```

Comment:
im not sure why it's blue, but when you hover over it should make the text yellow and the bg white

Reply:
Won't white bg dissapear into the white background?
![image](https://github.com/acm-ucr/hiss-website/assets/85457506/722947ca-fe5b-4687-b50e-ba0b42b3b0ab)

Reply:
yeah that's okay, i think the current blue doesn't look good